### ISSUE TITLE
My loss immediately dropped to zero and weights can't work

### ISSUE THREAD

**Issue by jack04060201** [AlexeyAB/darknet#6213]
Hello , I'm training on colab but my chart look so weird.
![chart](https://user-images.githubusercontent.com/17104362/87239225-32a6d700-c43f-11ea-8a14-a0b634076454.png)
It drop to zero very fast and weight that produce can't detect anything.
I check my file and path so many times but it still have problem.
I have 6 classes.
this is my file structure
![Untitled Diagram (2)](https://user-images.githubusercontent.com/17104362/87239249-95986e00-c43f-11ea-8c7f-d630fb6b88c5.png)

and use these code to train.
```
!./darknet partial cfg/yolov3_training.cfg yolov3-tiny.weights yolov3-tiny.conv.15 15 #初次使用時可先預訓練
!./darknet detector train data/obj.data cfg/yolov3_training.cfg yolov3-tiny.conv.15 -dont_show | grep "avg loss"
```
yolov3_training.cfg
```
[net]
# Testing
batch=64
subdivisions=64
# Training
# batch=64
# subdivisions=2
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 4000
policy=steps
steps=400000,450000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=33
activation=linear



[yolo]
mask = 3,4,5
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=6
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=33
activation=linear

[yolo]
mask = 0,1,2
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=6
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
```

obj.data:
```
classes=6
train  = data/train.txt
valid  = data/valid.txt
names = data/obj.names
backup = save
```
obj.names:
```
water
fire
wood
light
dark
heart
```
train.txt
```
train/img/0000013.jpg
train/img/0000017.jpg
train/img/0000015.jpg
train/img/0000014.jpg
train/img/0000019.jpg
train/img/0000018.jpg
train/img/0000016.jpg
train/img/0000011.jpg
train/img/0000010.jpg
train/img/0000008.jpg
train/img/0000012.jpg
train/img/0000002.jpg
train/img/0000001.jpg
train/img/0000005.jpg
train/img/0000004.jpg
train/img/0000009.jpg
train/img/0000007.jpg
train/img/0000006.jpg
train/img/0000003.jpg
train/img/0000000.jpg
```
bad.list like this
```
train/img/0000004.txt
train/img/0000007.txt
train/img/0000000.txt
train/img/0000006.txt
train/img/0000004.txt
train/img/0000010.txt
train/img/0000010.txt
train/img/0000012.txt
train/img/0000001.txt
train/img/0000001.txt
train/img/0000016.txt
train/img/0000016.txt
train/img/0000003.txt
train/img/0000019.txt
train/img/0000019.txt
train/img/0000014.txt
             .
             .
             .
train/img/0000000.txt
train/img/0000001.txt
```

```
CUDA-version: 10010 (10010), cuDNN: 7.6.5, GPU count: 1  
 OpenCV version: 3.2.0
 0 : compute_capability = 600, cudnn_half = 0, GPU: Tesla P100-PCIE-16GB 
   layer   filters  size/strd(dil)      input                output
```

**Comment by AlexeyAB** [comment]
* check your dataset - run training with flag `-show_imgs` i.e. `./darknet detector train ... -show_imgs` and look at the `aug_...jpg` images, do you see correct truth bounded boxes?
    * show content of generated files `bad.list` and `bad_label.list` if they exist

**Comment by jack04060201** [comment]
> ```
> * check your dataset - run training with flag `-show_imgs` i.e. `./darknet detector train ... -show_imgs` and look at the `aug_...jpg` images, do you see correct truth bounded boxes?
> * show content of generated files `bad.list` and `bad_label.list` if they exist
> ```

I found the problem , .txt should in the same folder with .jpg , thank you!
